nilearn/datasets/func.py

```diff
@@ -2098,5 +2111,18 @@ def fetch_development_fmri(n_subjects=None, data_dir=None, resume=True,
                                                            url=None,
                                                            verbose=verbose)
 
+    if reduce_confounds:
+        reduced_regressors = []
+        for in_file in regressors:
+            out_file = os.path.splitext(in_file)[0] + '_reduced.tsv'
```

Comment:
It would be more BIDS-compliant to change the `desc` value to `reducedConfounds`. No strong opinion, but one idea.